kwave/options/simulation_options.py

```diff
@@ -76,8 +77,8 @@ class SimulationOptions(object):
     radial_symmetry: str = 'WSWA-FFT'
```

Comment:
Could time-reversal also be added here?

Reply:
Flags like time-reversal, axisymmetric, cuboid-corners are determined based on the other fields set by user. What do you think about moving the code where the value of these flags are determined to the `SimulationOptions` class? By passing a few options (e.g. `sensor` or `sensor mask`), caller can determine the value of this field. If you agree, I can make the changes.

Reply:
I agree